src/js/components/sharedComponents/InfoContent.tsx

```diff
@@ -1,33 +1,175 @@
 import React, { FunctionComponent, useEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
+import convert from 'xml-js';
 
 import { RootState } from 'js/store';
 
 import { downloadData } from '../../../../configs/modal.config';
 
+interface MetadataContent {
+  title?: string;
+  functionOrPurpose?: string;
+  geographicCoverage?: string;
+  tags?: string;
+  overview?: string;
+}
+
+interface ContentError {
+  error: string;
+}
+
+type Content = MetadataContent | ContentError;
+
+const getMetadata = async (layer: any): Promise<any> => {
+  let content: any;
+  const metadataURL = `https://www.arcgis.com/sharing/rest/content/items/${layer.portalItemID}/info/metadata/metadata.xml`;
```

Comment:
the domain "https://www.arcgis.com" should be coming from the redux `appSettings.sharinghost` can you double check that it is the case in PROD as well?

Reply:
@vaidotasp confirmed - that's what's happening in `clean-slate`. I'll implementing this now